cpu/fe310/include/sifive/platform.h

```diff
@@ -0,0 +1,129 @@
+// See LICENSE for license details.
+
+#ifndef _SIFIVE_PLATFORM_H
+#define _SIFIVE_PLATFORM_H
+
+// Some things missing from the official encoding.h
+#define MCAUSE_INT         0x80000000
+#define MCAUSE_CAUSE       0x7FFFFFFF
+
+
+
+#include "sifive/aon.h"
+#include "sifive/clint.h"
+#include "sifive/gpio.h"
+#include "sifive/otp.h"
+#include "sifive/plic.h"
+#include "sifive/prci.h"
+#include "sifive/pwm.h"
+#include "sifive/spi.h"
+#include "sifive/uart.h"
+
+
+/****************************************************************************
+ * Platform definitions
+ *****************************************************************************/
+
+// Memory map
+#define MASKROM_MEM_ADDR 		(0x00001000)
+#define TRAPVEC_TABLE_CTRL_ADDR	(0x00001010)
+#define OTP_MEM_ADDR 			(0x00020000)
+#define CLINT_CTRL_ADDR 		(0x02000000)
+#define PLIC_CTRL_ADDR 			(0x0C000000)
+#define AON_CTRL_ADDR 			(0x10000000)
+#define PRCI_CTRL_ADDR 			(0x10008000)
+#define OTP_CTRL_ADDR 			(0x10010000)
+#define GPIO_CTRL_ADDR 			(0x10012000)
+#define UART0_CTRL_ADDR 		(0x10013000)
+#define SPI0_CTRL_ADDR 			(0x10014000)
+#define PWM0_CTRL_ADDR 			(0x10015000)
+#define UART1_CTRL_ADDR 		(0x10023000)
+#define SPI1_CTRL_ADDR 			(0x10024000)
+#define PWM1_CTRL_ADDR 			(0x10025000)
+#define SPI2_CTRL_ADDR 			(0x10034000)
+#define PWM2_CTRL_ADDR 			(0x10035000)
+#define SPI0_MEM_ADDR 			(0x20000000)
+#define MEM_CTRL_ADDR 			(0x80000000)
+
+// IOF masks
+#define IOF0_SPI1_MASK          (0x000007FC)
+#define SPI11_NUM_SS     		(4)
+#define IOF_SPI1_SS0          	(2u)
+#define IOF_SPI1_SS1          	(8u)
+#define IOF_SPI1_SS2          	(9u)
+#define IOF_SPI1_SS3          	(10u)
+#define IOF_SPI1_MOSI         	(3u)
+#define IOF_SPI1_MISO         	(4u)
+#define IOF_SPI1_SCK          	(5u)
+#define IOF_SPI1_DQ0          	(3u)
+#define IOF_SPI1_DQ1          	(4u)
+#define IOF_SPI1_DQ2          	(6u)
+#define IOF_SPI1_DQ3          	(7u)
+
+#define IOF0_SPI2_MASK          (0xFC000000)
+#define SPI2_NUM_SS       		(1)
+#define IOF_SPI2_SS0          	(26u)
+#define IOF_SPI2_MOSI         	(27u)
+#define IOF_SPI2_MISO         	(28u)
+#define IOF_SPI2_SCK          	(29u)
+#define IOF_SPI2_DQ0          	(27u)
+#define IOF_SPI2_DQ1          	(28u)
+#define IOF_SPI2_DQ2          	(30u)
+#define IOF_SPI2_DQ3          	(31u)
+
+//#define IOF0_I2C_MASK          (0x00003000)
```

Comment:
Should be removed if not needed

Reply:
Vendor provided files.  It looks like renaming the sifive folder to vendor will suppress warnings